cyclone/vignettes/Running.Rmd

```diff
@@ -129,6 +129,16 @@ Required columns:
 In most cases, `used_for_UMAP`, `used_for_clustering`, `used_for_scaffold` can be the same, but there may be cases where you might use a marker for clustering, but not for UMAP calculation, for example.
 
 How to create: Use `CyTOF_pipeline/make_marker_metadata_csv.R` script.
+  
+To invoke this script, it is run from the terminal/command line prompt, NOT
+in Rstudio. 
+
+`$ Rscript make_marker_metadata_csv.R -f fcs_file.fcs -o marker_metadata.csv`
+
+Above is an example invocation. You must update your path to wherever your
+copy of `make_marker_metadata_csv.R` is located, and supply the full file path
+and file name of your own FCS file to run. `marker_metadata.csv` is the 
```

Comment:
I think a pointer to `fcs_file.fcs` would make it more clear that that's where the file path/name go for people less comfortable in the command line.
 
Perhaps ", and in place of `fcs_file.fcs` supply the full file path and file name of one of your own FCS file."

Reply:
Love that!